hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherClient.java

```diff
@@ -148,10 +148,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        final StringBuilder builder = new StringBuilder("CypherClient{");
-        builder.append("userName='").append(userName).append('\'')
-               .append(", token='").append(token).append('\'').append('}');
+        String builder = "CypherClient{" + "userName='" + userName + '\'' +
+                         ", token='" + token + '\'' + '}';
 
-        return builder.toString();
+        return builder;
```

Comment:
ignore the clean for `StringBuilder` (due to perf influence, check if we could config the rule to ignore it)

we could remove the unless SB transfer if we could ensure it's the call frequency is **low**